rdkit/Chem/TorsionFingerprints.py

```diff
@@ -272,7 +272,7 @@ def CalculateTorsionLists(mol, maxDev='equal', symmRadius=2, ignoreColinearBonds
     # get the torsions
     tmp = []
     num = len(r)
-    maxdev = 180.0 * math.exp(-0.025 * (num - 14) * (num - 14))
+    maxdev = 180.0 if 14 <= num else 180.0 * math.exp(-0.025 * (num - 14) * (num - 14))
```

Comment:
I find the python ternary operator to be pretty unreadable relative to the minimal benefit that it provides. Please just use:
```suggestion
    if num > 14:
        maxdev = 180.0
    else:
         maxdev = 180.0 * math.exp(-0.025 * (num - 14) * (num - 14))
```

Reply:
> I find the python ternary operator to be pretty unreadable relative to the minimal benefit that it provides. Please just use:

Ok. I have fixed it according to your suggestion.

Reply:
> > I find the python ternary operator to be pretty unreadable relative to the minimal benefit that it provides. Please just use:
> 
> Ok. I have fixed it according to your suggestion.

Thanks!